images/dind/master/ovn-kubernetes-master.sh

```diff
@@ -15,6 +15,7 @@ function ovn-kubernetes-master() {
 
   local master_config="${config_dir}/master-config.yaml"
   cluster_cidr=$(python -c "import yaml; stream = file('${master_config}', 'r'); y = yaml.load(stream); print y['networkConfig']['clusterNetworks'][0]['cidr']")
+  svc_cidr=172.30.0.0/16
```

Comment:
We should actually pull this from master-config.yaml too:

`svc_cidr=$(python -c "import yaml; stream = file('${master_config}', 'r'); y = yaml.load(stream); print y['networkConfig']['serviceNetworkCIDR']")`

Same for the node config file too.

Reply:
fixed.